src/PhpSpec/ObjectBehavior.php

```diff
@@ -110,7 +110,7 @@ public function getWrappedObject()
      *
      * @param int|string $key
      */
-    public function offsetExists($key): bool
+    public function offsetExists($key): Subject
```

Comment:
This is incompatible with the parent interface, which may become stricter. It's perhaps better to remove the type hint

Reply:
Fixed.